presentation/common/src/presentation-common/PresentationRpcInterface.ts

```diff
@@ -205,8 +205,8 @@ export class PresentationRpcInterface extends RpcInterface {
   // TODO: need to enforce paging on this
   public async computeSelection(_token: IModelRpcProps, _options: SelectionScopeRpcRequestOptions, _ids: Id64String[], _scopeId: string): PresentationRpcResponse<KeySetJSON> { return this.forward(arguments); }
 
-  /** @alpha TODO: need to page results of this */
-  public async compareHierarchies(_token: IModelRpcProps, _options: PresentationDataCompareRpcOptions): PresentationRpcResponse<PartialHierarchyModificationJSON[]> { return this.forward(arguments); }
+  /** @alpha */
+  public async compareHierarchies(_token: IModelRpcProps, _options: PresentationDataCompareRpcOptions): PresentationRpcResponse<HierarchyCompareInfoJSON> { return this.forward(arguments); }
```

Comment:
I don't think we can do this even for `@alpha`, because it's an RPC function. I think the solution is:
- Increase minor RPC interface version. This will make sure newer frontends can't use older backends and we only have to worry about older frontends using newer backends. 
- Deprecate the `compareHierarchies` method.
- Add a new method for paged compare - `compareHierarchiesPaged`?

Reply:
Added `compareHierarchiesPaged` and deprecated `compareHierarchies`